ts/Series/Map/MapSeries.ts

```diff
@@ -765,7 +780,11 @@ class MapSeries extends ScatterSeries {
             */
             const scale = svgTransform.scaleX;
             const flipFactor = svgTransform.scaleY > 0 ? 1 : -1;
-            if (renderer.globalAnimation && chart.hasRendered) {
+            if (
+                renderer.globalAnimation &&
+                chart.hasRendered &&
+                !chart.disableTransform
```

Comment:
Extra properties on the already crowded Chart object should be avoided. Can we solve this another way? Suggestions:
* Set `globalAnimation` to false so that the _if_ fails? But that may have side effects.
* Set a property on MapView instead of Chart. But in that case, `disableTransform` is confusing because of double negation (`disableTransform = false` => better `allowTransform = true`). But it is not really the transform we're disabling/allowing is it? From the code it seems like it is the transform _animation_. So if I understand this correctly, I suggest `MapView.allowTransformAnimation`. Or maybe `MapSeries.allowTransformAnimation`.

Reply:
I have changed it to `MapView.allowTransformAnimation` because if it would be on series then after removing series during drill down/up the flag would be also removed.